pkg/reaper/resource.go

```diff
@@ -64,6 +65,12 @@ func NewReaper(
 			desiredReaper.Spec.UiUserSecretRef.Name = DefaultUiSecretName(kc.Name)
 		}
 	}
+	// If the cluster is already initialized and some DCs are flagged as stopped, we cannot achieve QUORUM in the
+	// cluster for Reaper's keyspace. In this case we simply skip schema migration, otherwise Reaper wouldn't be able to
+	// start up.
+	if kc.Status.GetConditionStatus(k8ssandraapi.CassandraInitialized) == corev1.ConditionTrue && kc.HasStoppedDatacenters() {
```

Comment:
I'm hesitant here. What conditions should we check to decide to skip schema migration?

Reply:
I deployed a cluster with Reaper `deploymentMode: SINGLE`. Reaper is deployed to dc1. I go back and forth stopping/starting dc2. It causes an update of the Reaper Deployment. That should be avoidable. I'm ok with fixing it in a follow up issue if you prefer since it is arguably an optimization.